src/node/dns.ts

```diff
@@ -1,30 +1,70 @@
-import { default as dnsUtil } from 'node-internal:dns';
 import * as errorCodes from 'node-internal:internal_dns_constants';
+import {
+  getServers,
+  lookup,
+  lookupService,
+  resolve,
+  resolve4,
+  resolve6,
+  resolveAny,
+  resolveCname,
+  resolveCaa,
+  resolveMx,
+  resolveNaptr,
+  resolveNs,
+  resolvePtr,
+  resolveSoa,
+  resolveSrv,
+  setDefaultResultOrder,
+  getDefaultResultOrder,
+  setServers,
+} from 'node-internal:internal_dns';
+import * as dns from 'node-internal:internal_dns';
 
 export * from 'node-internal:internal_dns_constants';
+export {
+  getServers,
+  lookup,
+  lookupService,
+  resolve,
+  resolve4,
+  resolve6,
+  resolveAny,
+  resolveCname,
+  resolveCaa,
+  resolveMx,
+  resolveNaptr,
+  resolveNs,
+  resolvePtr,
+  resolveSoa,
+  resolveSrv,
+  setDefaultResultOrder,
+  getDefaultResultOrder,
+  setServers,
+} from 'node-internal:internal_dns';
+
+function toCallback<A = unknown, ReturnValue = unknown>(
```

Comment:
You know we implement `callbackify`? You can move its implementation into `node/internal` and reuse that here.

Reply:
TIL. Thanks